cpp/src/merge/merge.cu

```diff
@@ -274,10 +270,12 @@ struct column_merger {
     //
     p_merged_col->set_null_count(lcol.null_count() + rcol.null_count());
 
+    using Type = get_column_stored_type<Element>;
+
     // to resolve view.data()'s types use: Element
     //
-    Element const* p_d_lcol = lcol.data<Element>();
-    Element const* p_d_rcol = rcol.data<Element>();
+    auto const p_d_lcol = lcol.data<Type>();
+    auto const p_d_rcol = rcol.data<Type>();
```

Comment:
We don't need too much Hungarian notation. (But  the `d_` is useful in CUDA code to represent device pointers when there is a host pointer of mostly the same name in the scope).

```suggestion
    auto const d_lcol = lcol.data<Type>();
    auto const d_rcol = rcol.data<Type>();
```